app/models/piece.rb

```diff
@@ -2,6 +2,11 @@ class Piece < ActiveRecord::Base
   belongs_to :user
   belongs_to :game
 
+  def move_piece(new_x, new_y)
```

Comment:
Done - merged it into @ruishine's `move_to!` method, which was already written in almost exactly the same way. Great minds think alike. ;)